pkg/wallet/wallet.go

```diff
@@ -151,7 +151,7 @@ func (w *BaseWallet) UnpackMessage(encMessage []byte) (*Envelope, error) {
 		if err != nil {
 			return nil, fmt.Errorf("failed from decrypt: %w", err)
 		}
-		return &Envelope{Message: bytes}, nil
+		return &Envelope{Message: bytes, ToVerKeys: []string{recipVKeyB58}}, nil
```

Comment:
`recipVKeyB58` is not the only recipient key in UnpackeMessage.. we need to get all recipient's keys.. suggest to create an issue for this to be done in a separate pr 

also Decrypt() is currently being refactored to not pass any key in the arguments list. so this code will change a bit (depends who's PR gets merged first)

Reply:
That was my concern too, as per current logic of `baseWallet.UnpackMessage()` we try to decrypt message from very first recipient key, if we can't decrypt then we throw error, if we can decrypt we return message. Looks odd, created #468

Reply:
yes, this logic will be changed in my PR.. we will find the first matching recipient key in the wallet from the list of recipients in the envelope:
https://github.com/hyperledger/aries-framework-go/pull/459/files#diff-e602f67bb01eb068d2869bf2d53256d4R93

so UnpackMessage() will not use the first recipient in the list anymore

Reply:
this is why, I believe `ToVerKeys` above should contain the list of recipients keys in the envelope, not just the current decrypting recipient key

Reply:
@sudeshrshetty @Baha-sk is there an action for this PR?

Reply:
no action in this PR, #468 will be addressed once Baha's changes for decrypt() gets merged